styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,77 @@
+a.footer-menu-link {
+  color: darken($light-gray, 15%);
+  font-size: 1.2rem;
+}
+a.footer-tab-link{
+  color:$white;
+  font-size: 1.3rem;
+}
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+}
+.footer-menu {
+  li {
+    list-style: none;
+    padding: 0;
+    margin: 8px 0;
+  }
+  &.split {
+    column-count: 3;
+  }
+}
+@media only screen and (max-width: $screen-sm) {
+  a.footer-menu-link{
+    display: block;
+    background-color: lighten($blue-gray, 05%);
+    padding:15px;
+    margin: 0;
+    &:hover{
+      background-color: lighten($blue-gray,08%);
+    }
+  }
+  a.footer-tab-link{
+    padding:15px;
+    display: block;
+    color:$white;
```

Comment:
You shouldn't have to redefine color here if it's already set as the default color for the same selector outside of the media query.